### PR TITLE
docs(update): Re-optimised the image sizes

### PR DESCRIPTION
**Pull request by barefoot-88**
Re-optimised the image sizes of coin-selection/funding-tx.jpg and @2x.jpg

Images are now 1200px and 2400px wide and should be suitable for desktop displays

### CODE REVIEW

**Comment by pavlenex** [comment]
@GBKS so the image is replaced, but it still kinda loooks the same, since you're a resolution expert 😄 , what's needed here?

Is it the markdown itself that should be modified?

```
{% include picture.html
   image = "/assets/images/guide/payments/coin-selection/funding-tx.jpg"
   retina = "/assets/images/guide/payments/coin-selection/funding-tx@2x.jpg"
   mobile = "/assets/images/guide/payments/coin-selection/funding-tx-mobile.jpg"
   mobileRetina = "/assets/images/guide/payments/coin-selection/funding-tx-mobile@2x.jpg"
   alt-text = "Flow chart showing how unspent outputs from a wallet form inputs of a new transaction and the contained bitcoin end up in the receiving wallet."
   width = 800
   height = 400
%}
```
Or is it the image?

**Comment by GBKS** [comment]
No probs. Just update the width and height attributes to the width and height of your newly exported image. Also add `layout = "full-width"`  as shown [here](https://bitcoin.design/guide/contribute/formatting/#images).

**Comment by pavlenex** [comment]
I did the changes @GBKS still looks a bit blurry, feel free to push a fix if there's one, it's better than before, but seems not as sharp, though it can be caching on my end, unsure.

#### Before
Screenshot 2021-08-10 at 21 12 08

#### After
Screenshot 2021-08-10 at 21 12 20

File: guide/payments/coin-selection.md [hunk 1]
```diff
@@ -33,8 +33,9 @@ There are two types of coin selection strategies that are used in Bitcoin applic
    mobile = "/assets/images/guide/payments/coin-selection/funding-tx-mobile.jpg"
    mobileRetina = "/assets/images/guide/payments/coin-selection/funding-tx-mobile@2x.jpg"
    alt-text = "Flow chart showing how unspent outputs from a wallet form inputs of a new transaction and the contained bitcoin end up in the receiving wallet."
-   width = 800
-   height = 400
+   width = 1200
```
Comment (Bosch-0) [inline guide/payments/coin-selection.md:36]: Try with these sizes. 

W: 2400px 
H: 1200px

Comment (GBKS) [reply guide/payments/coin-selection.md:36]: 1200 is correct. Needs to be the 1x image size.

**Review by GBKS — APPROVED** [review]

**Comment by Bosch-0** [comment]
Image still looks blurry to me on desktop.

**Comment by GBKS** [comment]
Took a closer look and there were duplicate images in incorrect folder structure floating around, which I deleted. The new images replaced in this PR were amongst those unused ones, so I copied the new images over to the correct folder. Both fixes are in #452 .

**Pull request merged**